account_move_line_purchase_info/models/purchase_order_line.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019-2020 ForgeFlow S.L.
+#   (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    @api.depends("order_line.invoice_lines.move_id")
+    def _compute_invoice(self):
+        """Overwritten compute to avoid show all Journal Entries with
+        purchase_order_line as invoice_lines One2many would take them into account."""
+        for order in self:
+            invoices = order.mapped("order_line.invoice_lines.move_id").filtered(
+                lambda m: m.is_invoice(include_receipts=True)
+            )
+            order.invoice_ids = [(6, 0, invoices.ids)]
```

Comment:
```suggestion
            order.invoice_ids = invoices
```
same but looks better :P

Reply:
@ivantodorovich Thanks for the review! Solved :)